FastSurferCNN/quick_qc.py

```diff
@@ -101,27 +100,27 @@ def check_volume(asegdkt_segfile, voxvol, thres=0.70):
 def get_region_bg_intersection_mask(
     seg_array, region_labels=VENT_LABELS, bg_label=BG_LABEL
 ):
-    """Return a mask of the intersection between the voxels of a given region and background voxels.
-    
+    """
+    Return a mask of the intersection between the voxels of a given region and background voxels.
+
     This is obtained by dilating the region by 1 voxel and computing the intersection with the
     background mask.
-    
+
     The region can be defined by passing in the region_labels dict.
 
     Parameters
     ----------
     seg_array : numpy.ndarray
-        Segmentation array
+        Segmentation array.
     region_labels : Dict
-        dict whose values correspond to the desired region's labels (Default value = VENT_LABELS)
+        Dict whose values correspond to the desired region's labels (Default value = VENT_LABELS).
     bg_label : int
-        (Default value = BG_LABEL)
+        (Default value = BG_LABEL).
```

Comment:
replace BG_LABEL with its value (e.g. make it a format str `f" bla {BG_LABEL} bla" `

Reply:
also should be in line above `bg_label : int, default={BG_LABEL}`